nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/pom.xml

```diff
@@ -139,6 +143,10 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-mail</artifactId>
+    </dependency>
```

Comment:
duplicate?

Reply:
Indeed, it's the test-jars I need.

I've already pushed.